README.md

```diff
@@ -203,6 +203,25 @@ There are multiple options where to get the _converter_ (it's the same case for
 1. Modules [extras-circe](extras-circe/README.md) and [extras-cactus](extras-cactus/README.md) provide support for JSON and GPB conversion. 
 1. Use `identity` converter by specifying `Bytes` type argument. No further action needed in that case.
 
+#### Consumer parsing failure
+
+It may happen the delivery is not parsable as your `MyClass`. It won't be passed into the consumer and `ParsingFailureAction` will be used
```

Comment:
I would prefer to have something like `PullResult` as input to regular consumer (created via `newConsumer`). So use a sealed trait instead of _plain_ `Delivery`. I believe the code would be more concise.